rules/recognizing-phishing-urls/rule.md

```diff
@@ -26,40 +26,46 @@ For the address <https://www.ssw.com.au/services>, services is the path. The pat
 
 You should always check that the **domain** matches the service or website you are expecting.
 
+::: greybox
 <http://linkedin.com.sggr.ru/someaddress>
```

Comment:
We don't want clickable links in these examples, because they will be reported as broken.

Do you know how to make them not clickable? 
Tip: Use HTML characters. Ping me if in doubt